launcher/app.py

```diff
@@ -33,6 +33,10 @@ def __init__(self, root, source):
         except IOError:
             raise  # Server refused to connect
 
+        # Install actions
+        actions.register_default_actions()
```

Comment:
Now this has moved into the App's `__init__` it should also technically need a `deregister` method for it. Otherwise it will end up having more and more of the same actions registered when initializing Application more than once.

These might make more sense to just be globally registered once since they are **always** present.

Reply:
This is indeed the easier approach so I am in favor. I do want to note that with this we do have to rely on `os.environ` to fetch the `AVALON_CONFIG` instead `Session`.

The register functions would be called in the launcher's `__init__` module as previously

Reply:
Ah, no. You're talking about something else - that's the "config" actions. I'm talking about the default actions belonging to the launcher those are easier to refactor right?

The ones from the config indeed are open to discussion, but that was so in the issue too, right?